misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathAssertionsSpec.kt

```diff
@@ -689,6 +748,35 @@ abstract class PathAssertionsSpec(
             expect(folder).isDirectoryFun()
         }
     }
+
+    describeFun(hasSameBinaryContentAs.name) {
+        val hasSameBinaryContentAsFun = hasSameBinaryContentAs.lambda
+
+        context("hasSameBinaryContentAs") {
+            it("do not throw assertion error when have same binary content") withAndWithoutSymlink { maybeLink ->
```

Comment:
once you describe both functions in the same describe then change this to:
```suggestion
        context("has same binary content") {
            it("${hasSameBinaryContentAs.name} - does not throw") withAndWithoutSymlink { maybeLink ->
```
and then I would like to see the following test cases for hasSameTextualContent:
- does not throw if UTF-8, UTF-8 is used
- does not throw if UTF-16, UTF-16 is used
- throws AssertionError if UTF-8, UTF-16 is used

Moreover I would like to see the following context: `same text but UTF-8 and UTF-16 encoding`:
- hasSameBinaryContentAs - throws AssertionError
- hasSameTextualContent - does not throw if UTF-8, UTF-16 is used
- throws AssertionError if UTF-8, UTF-8 is used

Reply:
When I try to write test that throws exception, tests fail as NoSuchFile exception is thrown and i could not get what i was doing wrong